internal/controller/nim/account_controller.go

```diff
@@ -99,6 +102,21 @@ func (r *AccountReconciler) SetupWithManager(mgr ctrl.Manager, ctx context.Conte
 				}
 				return requests
 			})).
+		WithEventFilter(predicate.Funcs{
+			UpdateFunc: func(e event.UpdateEvent) bool {
```

Comment:
are you sure you only need the update event for the object? with this, I believe create/delete will be ignrored.

Reply:
Are you sure? I thought it was the other way around.
When #380 gets merged, I'll rebase and verify.

Reply:
https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.1/pkg/predicate#TypedPredicate
if it is only overwritten Update() here, others should preserve the default logic

Reply:
yes, AFAIK it will not touch the other logic.

Reply:
I tested this using an intermediate image in our development environment; it works as expected. I'm only filtering update events; other events work without any filtering.

Reply:
ok my bad :)